commerce/src/inline-price.js

```diff
@@ -7,18 +7,91 @@ import {
 import { selectOffers, useService } from './utilities.js';
 import { GeoMap } from './settings';
 
+const INDIVIDUAL = 'INDIVIDUAL_COM';
+const BUSINESS = 'TEAM_COM';
+const STUDENT = 'INDIVIDUAL_EDU';
+const UNIVERSITY = 'TEAM_EDU';
+
 // countries where tax is displayed for all segments by default
 const DISPLAY_ALL_TAX_COUNTRIES = [GeoMap.uk, GeoMap.au, GeoMap.fr, GeoMap.at, GeoMap.be_en, GeoMap.be_fr, GeoMap.be_nl, GeoMap.bg, GeoMap.ch_de, GeoMap.ch_fr, GeoMap.ch_it,
     GeoMap.cz, GeoMap.de, GeoMap.dk, GeoMap.ee, GeoMap.eg_ar, GeoMap.eg_en, GeoMap.es, GeoMap.fi, GeoMap.fr, GeoMap.gr_el, GeoMap.gr_en, GeoMap.hu, GeoMap.ie, GeoMap.it, GeoMap.lu_de,
     GeoMap.lu_en, GeoMap.lu_fr, GeoMap.nl, GeoMap.no, GeoMap.pl, GeoMap.pt, GeoMap.ro, GeoMap.se, GeoMap.si, GeoMap.sk, GeoMap.tr, GeoMap.ua, GeoMap.id_en, GeoMap.id_id,
     GeoMap.in_en, GeoMap.in_hi, GeoMap.jp, GeoMap.my_en, GeoMap.my_ms, GeoMap.nz, GeoMap.th_en, GeoMap.th_th];
 // countries where tax is displayed for some segments only by default
 const DISPLAY_TAX_MAP = {
-    'INDIVIDUAL_COM': [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.sa_ar, GeoMap.sa_en, GeoMap.za, GeoMap.sg, GeoMap.kr], // individual
-    'TEAM_COM': [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.za, GeoMap.co, GeoMap.kr], // business
-    'INDIVIDUAL_EDU': [GeoMap.lt, GeoMap.lv, GeoMap.sa_en, GeoMap.sea], // student
-    'TEAM_EDU': [GeoMap.sea, GeoMap.kr], // school and uni
+    [INDIVIDUAL]: [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.sa_ar, GeoMap.sa_en, GeoMap.za, GeoMap.sg, GeoMap.kr],
+    [BUSINESS]: [GeoMap.za, GeoMap.lt, GeoMap.lv, GeoMap.ng, GeoMap.za, GeoMap.co, GeoMap.kr],
+    [STUDENT]: [GeoMap.lt, GeoMap.lv, GeoMap.sa_en, GeoMap.sea],
+    [UNIVERSITY]: [GeoMap.sea, GeoMap.kr],
+};
+
+// For most countries where tax label is displayed the tax is included for Individuals and Students
+// and excluded for Business and Universities. This is the map of TaxExclusive values for other countries
+const TAX_EXCLUDED_MAP = {
+    [GeoMap.za]: [false, false],
```

Comment:
what will happen for GeoMap.za in segment 'STUDENT'? (in case it will be created in the future)
just wondering if we could have all values with all 4 segments, or its not possible

Reply:
I filled the table with Defaults.forceTaxExclusive. It would return that value even without this change but it is more obvious like this. Also Instead of false I use now Defaults.forceTaxExclusive and Defaults.displayTax in inline-price.js